rpcs3/Emu/Cell/SPURecompiler.cpp

```diff
@@ -7311,6 +7327,32 @@ class spu_llvm_recompiler : public spu_recompiler_base, public cpu_translator
 
 		if (g_cfg.core.spu_approx_xfloat)
 		{
+			if (auto cv = llvm::dyn_cast<llvm::Constant>(b.value))
```

Comment:
Could be done in all paths including accurate xfloat.

Reply:
Also might want to check the first operand as well for a reverse optimization.

Reply:
It's not that simple, accurate xfloat path may not benefit from this in case it'll force conversion from f64 to f32.

Reply:
Can't be ported to accurate xfloat path with s64[4] comparison instead?

Reply:
But It can also be done later.